VCS/1. User Guides/6 連線 Windows 個體：本機為 Windows.md

```diff
@@ -1 +1,48 @@
+---
+tags: Guide, VCS, TW
+title: 連線 Windows：本機為 Windows | en
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+
+# Connect to Windows instance: local machine as Windows
+
+After finished [Preparation and Obtain connection information](https://man.twcc.ai/@twccdocs/vcs-guide-connect-prerequisite-en), please refer to the following method to connect to the Windows instance ( local machine as Windows).
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+Please note that the status of the instance must be **`Ready`** before it can be used online.
+:::
+
+:::danger
+{%hackmd @docsharedstyle/important-en %}
+1. Due to frequent information security incidents, if your connection comes from the following countries, we will disable your remote connection to Windows instance (port: 9833): China, Germany, France, South Korea, Netherlands, Poland, Russia. If you need to connect to TWCC Windows instance, please contact customer service.
+2. If you modify the network interface card settings, you will not be able to connect to the Virtual Compute Service(VCS) instance. Therefore, we strongly recommend you to not change the network interface card settings. Please pay more attention during your operation and deployment.
```

Comment:
[SUG]

You cannot connect to VCS instance once the settings of the network interface card are modified. Therefore, we strongly recommend that you do not change the network interface card settings. Please beware when operating and deploying.